state/backend.go

```diff
@@ -57,6 +59,24 @@ func (st *State) localID(id string) string {
 	return localID
 }
 
+// ParseLocalIDToTags tries to parse a DocID e.g. `c9741ea1-0c2a-444d-82f5-787583a48557:a#mediawiki
+// to a corresponding Tag. In the above case -> applicationTag.
+func (st *State) ParseLocalIDToTags(docID string) names_v3.Tag {
```

Comment:
I was not sure which place was perfect for this. 
Thus I added this close to the place which already had some util functions.

Reply:
And I am not sure if that are all the important Tags..

Reply:
@manadart do you think this catches all the possible docID kinds (for constraints) we have? 
I am not sure which could be missing.

@mitechie  ^